packages/p2p-v2/src/pages/my-ads/screens/MyAds/MyAdsTable/MyAdsDisplayWrapper.tsx

```diff
@@ -10,11 +12,17 @@ type TMyAdsDisplayWrapperProps = {
 
 const MyAdsDisplayWrapper = ({ children, isPaused, onClickToggle }: PropsWithChildren<TMyAdsDisplayWrapperProps>) => {
     const { isMobile } = useDevice();
+    const history = useHistory();
     if (isMobile) {
         return (
             <FullPageMobileWrapper
                 renderFooter={() => (
-                    <Button isFullWidth size='lg' textSize='md'>
+                    <Button
+                        isFullWidth
+                        onClick={() => history.push(`${BASE_URL}/my-ads/create`)}
```

Comment:
can we use this Constant instead, since you have added it already 🙏 

```suggestion
                        onClick={() => history.push(`${MY_ADS_URL}/create`)}
```